### PR TITLE
sync readme with docs site

### PR DESCRIPTION
**Pull request by dB2510**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,34 +20,49 @@ In the future, this repo aims to contain compose files for every possible Execut
...
-# Deletes previously created cluster
-make clean
+   ```sh
+   # Copy the sample environment variables
+   cp .env.sample .env
+   ```
 
-# Create the artifacts for a new test cluster
-make create
+   For simplicities sake, this repo is configured to work with a remote Beacon node such as one from [Infura](https://infura.io/).
```
Comment (xenowits) [inline README.md:40]:
```suggestion
   For simplicity's sake, this repo is configured to work with a remote Beacon node such as one from [Infura](https://infura.io/).
```

File: README.md [hunk 2]
```diff
@@ -20,34 +20,49 @@ In the future, this repo aims to contain compose files for every possible Execut
...
 
-# Start the cluster
-make up
+   Create an Eth2 project and copy the `https` URL:
```
Comment (xenowits) [inline README.md:42]: also, add a note that `prater` is selected in endpoint

File: README.md [hunk 3]
```diff
@@ -20,34 +20,49 @@ In the future, this repo aims to contain compose files for every possible Execut
...
-# Open Jaeger dashboard
-open http://localhost:16686
-```
+   Replace the placeholder value of `CHARON_BEACON_NODE_ENDPOINT` in your newly created `.env` file with this URL.
+
+1. Create the artifacts needed to run a testnet distributed validator cluster
+
+   ```sh
+   # Create a testnet distributed validator cluster
+   docker run --rm -v "$(pwd):/opt/charon" ghcr.io/obolnetwork/charon:latest create cluster --cluster-dir=".charon" --withdrawal-address="0x000000000000000000000000000000000000dead"
```
Comment (xenowits) [inline README.md:52]: can remove `--cluster-dir` flag

File: README.md [hunk 4]
```diff
@@ -20,34 +20,49 @@ In the future, this repo aims to contain compose files for every possible Execut
...
+   ```
+
+1. Start the cluster
```
Comment (xenowits) [inline README.md:55]: change numbering

File: README.md [hunk 5]
```diff
@@ -20,34 +20,49 @@ In the future, this repo aims to contain compose files for every possible Execut
...
-# Open Jaeger dashboard
-open http://localhost:16686
-```
+   Replace the placeholder value of `CHARON_BEACON_NODE_ENDPOINT` in your newly created `.env` file with this URL.
+
+1. Create the artifacts needed to run a testnet distributed validator cluster
+
+   ```sh
+   # Create a testnet distributed validator cluster
+   docker run --rm -v "$(pwd):/opt/charon" ghcr.io/obolnetwork/charon:latest create cluster --cluster-dir=".charon" --withdrawal-address="0x000000000000000000000000000000000000dead"
```
Comment (xenowits) [inline README.md:52]: should we remove latest flag?

**Review by xenowits — APPROVED** [review]

**Comment by thomasheremans** [comment]
Just testing again and using the `make` commands currently in the readme starts the cluster successfully while the `docker compose up` instructions of the docs doesn't. Unsure what the issue is.

**Comment by xenowits** [comment]
Have you tested after checking out this branch?

**Pull request merged**